utils/utils.go

```diff
@@ -66,3 +66,11 @@ func AddAnnotation(obj metav1.Object, key string, value string) {
 func GetRegexMatcher(alerts []string) string {
 	return "^" + strings.Join(alerts, "$|^") + "$"
 }
```

Comment:
Why is this func in the general utils module? This seems like something that need to be added to the specific template that uses it

Reply:
This is used for alertmanager configs, unrelated to the PR. Do I make the changes in this PR or a seperate one?

Reply:
Used here
https://github.com/red-hat-storage/ocs-osd-deployer/blob/158e9723b18bf1914cc79e44122eb310d7ef7e84/templates/alertmanagerconfig.go#L82

Reply:
As discussed, modified similar changes for alertmanager config as well